lib/ui/page/home/page/user/controller.dart

```diff
@@ -84,6 +85,9 @@ class UserController extends GetxController {
   /// [inContacts] indicator.
   StreamSubscription? _contactsSubscription;
 
+  /// [StreamSubscription] to the [RxUser.updates] of this [user].
+  StreamSubscription? _userSubscription;
```

Comment:
This seems like a hack, rather than the problem solution. Why we need to created additional fields, why not just handle this inside `RxUser` itself?

```dart
user?.listenUpdates();
user?.stopUpdates();
```